src/App.jsx

```diff
@@ -4,19 +4,21 @@ import React, { Component, Suspense } from 'react';
 import FullStory from 'react-fullstory';
 import ReactGA from 'react-ga4';
 import TagManager from 'react-gtm-module';
-import { Redirect, Route, Switch } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Redirect, Route, Switch, withRouter } from 'react-router-dom';
 import styled from 'styled-components';
 import muiTheme from './js/common/components/Style/muiTheme';
-import ErrorBoundary from './js/common/components/Widgets/ErrorBoundary';
 import LoadingWheelComp from './js/common/components/Widgets/LoadingWheelComp';
-import WeVoteRouter from './js/common/components/Widgets/WeVoteRouter';
+import historyPush from './js/common/utils/historyPush';
```

Comment:
@SailingSteve can you think of any reason moving "WeVoteRouter" to the src/index.jsx would cause problems, either in WebApp or Cordova?